DAO.sol

```diff
@@ -715,6 +717,45 @@ contract DAO is DAOInterface, Token, TokenCreation {
         return true;
     }
 
+    function collectInheritedDAORewards(bool _toMembers) external noEther returns (bool _success) {
+        if (msg.sender != address(this))
+            throw;
+
+        address daoAddress = privateCreation;
+
+        while (daoAddress != 0) {
+            DAO parentDAO = DAO(daoAddress);
+            parentDAO.retrieveDAOReward(_toMembers);
+            daoAddress = parentDAO.privateCreation();
+        }
+
+        return true;
+    }
+
+    function transferRewardTokens(address _to, uint _amountNum, uint _amountDen) external noEther returns (bool _success) {
+
+        if (   (_amountDen == 0)
+            || (_amountNum > _amountDen))
+            throw;
+        uint rewardTokensToTransfer = rewardToken[msg.sender] * _amountNum / _amountDen;
+
+        rewardToken[msg.sender] -= rewardTokensToTransfer;
```

Comment:
this needs to be called by the motherDAO, why should they do it? Theres is no incentivization for it.